scaffolds/cpp/Makefile

```diff
@@ -0,0 +1,32 @@
+cxxtestgen := cxxtestgen
```

Comment:
why cxxtest? why not boost.test or google test?

according its community, the activity of dev, and its philosophy (class based test), I think it's not a good choice.

Reply:
OK. I didn't know.

We made this choice during a code retreat where we neded something light that would work quickly. If it's the wrong choice we'll have to change.